src/elm/Pages/Build/View.elm

```diff
@@ -279,15 +280,29 @@ viewPreview msgs openMenu showMenu now zone org repo build =
                 [ text <| "#" ++ buildNumber ]
             ]
 
-        age =
-            [ text <| relativeTime now <| Time.millisToPosix <| Util.secondsToMillis build.created ]
-
         buildCreatedPosix =
             Time.millisToPosix <| Util.secondsToMillis build.created
 
+        age =
+            relativeTime now <| buildCreatedPosix
+
         timestamp =
             Util.humanReadableDateTimeFormatter zone buildCreatedPosix
 
+        displayTime =
+            if showTimestamp then
+                [ text <| timestamp ++ " " ]
+
+            else
+                [ text age ]
+
+        hoverTime =
+            if showTimestamp then
+                toString age
```

Comment:
the [Debug](https://package.elm-lang.org/packages/elm/core/latest/Debug) package is not available for use in production. 

is there an alternative to `toString` we can use? 
what is this function accomplishing?